test-suite-ex/src/test/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -10,6 +10,7 @@ package org.scalajs.testsuite.utils
 /** This is a partial copy of the implementation in the testSuite */
 object Platform {
 
+  def executingInNodeJS: Boolean = sysProp("nodejs")
   def typedArrays: Boolean = sysProp("typedarray")
 
   private def sysProp(key: String): Boolean =
```

Comment:
Really? I didn't mean `testSuiteEx.dependsOn(testSuite % "test")`. That would not be a good idea. But `Platfrom` should probably be in `main` anyways, no?